cmd/schedule-builder/README.md

```diff
@@ -0,0 +1,119 @@
+| 1.17.7        | 2020-06-12                                                                            | 2020-06-17  |
+| 1.17.6        | 2020-05-15                                                                            | 2020-05-20  |
+| 1.17.5        | 2020-04-13                                                                            | 2020-04-16  |
+| 1.17.4        | 2020-03-09                                                                            | 2020-03-12  |
+| 1.17.3        | 2020-02-07                                                                            | 2020-02-11  |
+| 1.17.2        | No-op release https://groups.google.com/d/topic/kubernetes-dev/Mhpx-loSBns/discussion | 2020-01-21  |
+| 1.17.1        | 2020-01-10                                                                            | 2020-01-14  |
+
+### 1.16
+
+Next patch release is **1.16.11**
+
+End of Life for **1.16** is **TBD**
+
+| PATCH RELEASE |                                 CHERRY PICK DEADLINE                                  | TARGET DATE |
+|---------------|---------------------------------------------------------------------------------------|-------------|
+| 1.16.11       | 2020-06-12                                                                            | 2020-06-17  |
+| 1.16.10       | 2020-05-15                                                                            | 2020-05-20  |
+| 1.16.9        | 2020-04-13                                                                            | 2020-04-16  |
+| 1.16.8        | 2020-03-09                                                                            | 2020-03-12  |
+| 1.16.7        | 2020-02-07                                                                            | 2020-02-11  |
+| 1.16.6        | No-op release https://groups.google.com/d/topic/kubernetes-dev/Mhpx-loSBns/discussion | 2020-01-21  |
+| 1.16.5        | 2020-01-10                                                                            | 2020-01-14  |
+| 1.16.4        | 2019-12-06                                                                            | 2019-12-11  |
+| 1.16.3        | 2019-11-08                                                                            | 2019-11-13  |
+| 1.16.2        | 2019-10-11                                                                            | 2019-10-15  |
+| 1.16.1        | 2019-09-27                                                                            | 2019-10-02  |
+```
+
+Also can save the schedule in a file, to do that, you can set the `--output-file` flag together with the filename.
+
+```
+$ schedule-builder --schedule-path ../sig-release/releases/schedule.yaml --output-file my-schedule.md
+```
+
+## Building From Source
+
+To build the `schedule-builder` tool, check out this repo to your `$GOPATH`:
+
+```
+git clone git@github.com:kubernetes/release.git $(go env GOPATH)/src/k8s.io/release
```

Comment:
Let’s either prefix all shell commands with `$ ` or none of them. :)

Reply:
done